packages/uploadthing/server.ts

```diff
@@ -1,4 +1,17 @@
+import { createNextRouteHandler } from "./next";
```

Comment:
Feels a bit backwards importing the nextjs adapter into here - seems like an abstraction may be in place?

Out of scope of this PR though

Reply:
I managed to get it working with just 1 more line in the handler, it shouldn't affect anything it should just work within solid and stuff like that:

https://github.com/pingdotgg/uploadthing/pull/41/commits/f5d1b276349f3c3b0fee985614a90cb877b4ef14#diff-5616e8338a9d8b19d93d8bb22aef8bfabe8e0557595f7f3d655fdffe00e99634R17